tests/components/mqtt/test_climate.py

```diff
@@ -1871,24 +1876,7 @@ async def test_temperature_unit(
                     },
                 ),
             ),
-            UnitOfTemperature.KELVIN,
```

Comment:
Note: KELVIN is not a valid value for unit system temperature unit, so I don't think it makes sense to try to keep this test

Reply:
Agree